.github/workflows/add-remove-labels.yml

```diff
@@ -31,19 +36,42 @@ jobs:
           comment-id: ${{ github.event.comment.id }}
           reactions: '+1'
 
+      # This currently fails with either the bot PAT or the standard github token secret
+      # gh: Insufficient scopes for reacting to this Pull Request Review Comment. (HTTP 403)
+      # {"message":"Insufficient scopes for reacting to this Pull Request Review Comment.","documentation_url":"https://docs.github.com/rest/reactions/reactions#create-reaction-for-a-pull-request-review-comment","status":"403"}
+      # It could work if we had a token with the proper permissions.
+      # See https://github.com/peter-evans/create-or-update-comment/issues/392 for why the action above doesn't work.
+      # Confirmed as a bug, see: https://github.com/github/docs/issues/36899
+      # - name: Acknowledge the review with thumbs up reaction
+      #   if: ${{ github.event.review }}
+      #   env:
+      #     GH_TOKEN: ${{ secrets.OPENDATAHUB_TESTS_BOT_PAT }}
+      #     REVIEW_COMMENT_ID: ${{ github.event.review.id }}
+      #     REPO_NAME: ${{ github.event.repository.name }}
+      #     REPO_OWNER: ${{ github.event.repository.owner.login }}
+      #   run: |
+      #     gh api \
+      #     --method POST \
+      #     -H "Accept: application/vnd.github+json" \
+      #     -H "X-GitHub-Api-Version: 2022-11-28" \
+      #     /repos/$REPO_OWNER/$REPO_NAME/pulls/comments/$REVIEW_COMMENT_ID/reactions \
+      #     -f "content=+1"
+
       - uses: actions/checkout@v4
 
       - name: Install uv
         uses: astral-sh/setup-uv@v5
 
       - name: Run add remove labels
         env:
-          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+          # GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
```

Comment:
please remove if not used

Reply:
if we agree on using the bot account rather than the actions bot yeah, I can remove it

Reply:
Removed